File: app/src/main/java/ru/aston/news/presenters/travel/TravelView.java
package ru.aston.news.presenters.travel;

import java.util.List;

import moxy.MvpView;
import ru.aston.news.dto.Post;


public interface TravelView extends MvpView {
     void updateToPost(List<Post> posts) ;

     void navigToPost(int id);
}

File: app/src/main/java/ru/aston/news/presenters/travel/TravelView.kt
package ru.aston.news.presenters.travel

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndStrategy
import moxy.viewstate.strategy.StateStrategyType
import ru.aston.news.dto.Post

@StateStrategyType(value = AddToEndStrategy::class)
interface TravelView : MvpView {
    fun updateRecycler(posts: List<Post>)

    fun showProgress()

    fun error()

}